### PR TITLE
fix media modal new folder button

### PR DESCRIPTION
**Pull request by djamnazi**
[OO-FEATURE] fix the new folder button on media modal

### CODE REVIEW

File: MediaAdminBundle/Resources/public/less/mediaModalFolder.less [hunk 1]
```diff
@@ -12,6 +12,10 @@
             ul li::before {
                 border-top: none;
             }
+
+            .media-modal-menu-new-folder .btn {
+                margin: 3px 0px !important;
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/less/mediaModalFolder.less]: !important it is not necessary

**Comment by djamnazi** [comment]
@alavieille  PR Updated

**Comment by alavieille** [comment]
:+1:

**Comment by ngilain** [comment]
:+1:

**Pull request merged**